app/simulator/domain/portfolio.py

```diff
@@ -0,0 +1,15 @@
+from simulator.domain.model import Trade
+
+
+class Portfolio:
+    def __init__(self) -> None:
+        self._tradelist: list[Trade] = []
+
+    def __repr__(self) -> str:
+        s = []
+        for trade in self._tradelist:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)